File: JAVA_SIS104/app/src/main/java/com/example/prueba_java/backend/WSdocumentos.java
package com.example.prueba_java.backend;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
public class WSdocumentos {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("nombre")
    @Expose
    private String nombre;
    @SerializedName("categoria")
    @Expose
    private String categoria;
    @SerializedName("fecha_creacion")
    @Expose
    private String fechaCreacion;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCategoria() {
        return categoria;
    }

    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }

    public String getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(String fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }
}

File: KOTLIN_SIS104/app/src/main/java/com/example/prueba_kotlin/backend/WSDocumentos.kt
package com.example.prueba_kotlin.backend

class WSDocumentos {
    @com.google.gson.annotations.SerializedName("id")
    @com.google.gson.annotations.Expose
    private var id: Int? = null

    @com.google.gson.annotations.SerializedName("nombre")
    @com.google.gson.annotations.Expose
    private var nombre: String? = null

    @com.google.gson.annotations.SerializedName("categoria")
    @com.google.gson.annotations.Expose
    private var categoria: String? = null

    @com.google.gson.annotations.SerializedName("fecha_creacion")
    @com.google.gson.annotations.Expose
    private var fechaCreacion: String? = null

    fun getId(): Int? {
        return id
    }

    fun setId(id: Int?) {
        this.id = id
    }

    fun getNombre(): String? {
        return nombre
    }

    fun setNombre(nombre: String?) {
        this.nombre = nombre
    }

    fun getCategoria(): String? {
        return categoria
    }

    fun setCategoria(categoria: String?) {
        this.categoria = categoria
    }

    fun getFechaCreacion(): String? {
        return fechaCreacion
    }

    fun setFechaCreacion(fechaCreacion: String?) {
        this.fechaCreacion = fechaCreacion
    }
}